File: CH_03_02.js
const readline = require("readline");

const rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout,
});

rl.question("Pick any number:", function (num) {
  if (num < 0) {
    console.log("Don't be so negative...");
  } else if (num < 10) {
    console.log("Between 0-10. Fair enough.");
  } else if (num < 1000001) {
    console.log("Good attitude.");
  } else {
    console.log("Don't be greedy.");
  }
  process.exit(0);
});

File: CH_03_02.py
number = int(input("Pick any number:"))

if number < 0:
    print("Don't be so negative...")
elif number < 10:
    print("Between 0-10. Fair enough.")
elif number < 1_000_001:
    print("Good attitude.")
else:
    print("Don't be greedy.")
